presto-client/src/main/java/io/prestosql/client/OkHttpUtil.java

```diff
@@ -125,6 +130,42 @@ private static InetSocketAddress toUnresolvedAddress(HostAndPort address)
         return InetSocketAddress.createUnresolved(address.getHost(), address.getPort());
     }
 
+    public static void setupInsecureSsl(
+            OkHttpClient.Builder clientBuilder)
+    {
+        try {
+            TrustManager trustAllCerts = new X509TrustManager()
+            {
+                @Override
+                public void checkClientTrusted(X509Certificate[] x509Certificates, String s) throws CertificateException
+                {
+                }
```

Comment:
Let's make this throw, since we don't expect it to be used
```java
throw new UnsupportedOperationException("checkClientTrusted should not be called");